### PR TITLE
Pull Request template

### PR DESCRIPTION
**Pull request by marijnschilling**
I wanted to give the PR template mentioned in [this](http://adavis.info/2018/09/frustration-free-code-reviews.html) article a try, but apparently it will only show up when its merged to the main branch. So we'd first have to merge it to see how it will work 😬 

I added it to the XCode project so you can easily edit the template rendered in markdown 💅  To make it render [this](https://dev.to/danielinoa_/rendering-markdown-in-xcode-9) post said i had to add some `.xcodesamplecode.plist` file. But please correct me if i'm wrong 😬

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- [ ] Bug fix 
+- [ ] This change requires a documentation update
+
+## Proof that it works
+
+Add screenshots or describe the tests that you ran to verify your changes.
+
+## Checklist:
+
+- [ ] I reviewe it myself
```
Comment (designatednerd) [inline .github/PULL_REQUEST_TEMPLATE.md]: `reviewe` -> `reviewed`

Comment (marijnschilling) [reply .github/PULL_REQUEST_TEMPLATE.md]: ow yes thanks! Actually already fixed that just forgot to push. That happens when you leave things for over the weekend 🙈

**Comment by designatednerd** [comment]
Taking another look at the `xcodesamplecode` thing - I don't think we necessarily want that here since we want to be able to edit this and/or other things that are in Markdown in Xcode, such as the README. I think getting it to render means that it's not editable, so I would recommend removing that file.

**Comment by marijnschilling** [comment]
Hmmm i actually added it so it would be easier to check for mistakes in the markdown file, you can just open the doc locally and edit in sublime or whatever text editor, and then see i Xcode if it looks e-oke. So you don't have to use an external markdown editor, or have to push and check on github to check wether its ok.

**Comment by marijnschilling** [comment]
Allright, not sure if its actually as convenient as i expect it to be, but if it aint I will change it. Shipping!

**Pull request merged**